docs/assets/includes/status-checks-important.md

```diff
@@ -5,8 +5,7 @@
     -   [Add coverage to your repository](../../coverage-reporter/index.md)
     -   Enable the rule **Diff coverage is under** or **Coverage variation is under** on the [pull request quality gate](../../repositories-configure/adjusting-quality-settings.md#gates).
 
-    <!--TODO Reword the sentence linking directly to the "Blocking pull requests section"-->
-    **To block merging pull requests** that aren't up to standards see [Integrating Codacy with your Git workflow](../../getting-started/integrating-codacy-with-your-git-workflow.md)
+    You can then optionally [block merging pull requests that aren't up to standards](../../getting-started/integrating-codacy-with-your-git-workflow.md#blocking-pull-requests).
```

Comment:
I worded this as a subsequent step after the user does the tasks above, although there are other ways besides coverage to block pull requests. I believe that this is OK, given the clear context.

Reply:
The way you reworded the sentence looks good :ok_hand: but now that you mention this, I find it quite weird that the admonition which had the main goal of instructing how to obtain status checks for coverage was also referencing the blocking of pull requests, since the two things are not necessarily dependent on one another. :thinking: 

While checking out the rendered page to see everything in context, I believe that the best place for this sentence is actually right after the explanation of what the status checks are, like this:

![image](https://user-images.githubusercontent.com/60105800/213639483-3438dce5-5ec8-4e9b-908e-d8f88e2dc3f0.png)

This has the advantage of giving a bit more visibility to the fact that status checks optionally allow blocking the merge of pull requests, and keeping this reference separated from the coverage-related admonition.

Reply:
Sopunds good. This also simplifies the import a bit, getting rid of the conditional.

Reply:
I hadn't thought of that, but that's a nice simplification indeed! :star_struck: